### PR TITLE
Bug 1305826, edited commands for creating new databases for MySQL, PostgresSQL, MongoDB

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1305826

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@vikram-redhat PTAL. Thanks!

**Comment by ahardin-rh** [comment]
@adellape PTAL. Per our conversation, added ifdef statements so that both OSE and Origin users get the right syntax.

**Comment by adellape** [comment]
@ahardin-rh You should be able to do ifdefs on lines within the code block, so that you don't have to repeat the full block. For example, [here](https://github.com/openshift/openshift-docs/blame/master/install_config/install/advanced_install.adoc#L502-L507).

**Comment by ahardin-rh** [comment]
Fixed. Thanks, @adellape !

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -70,10 +70,14 @@ MongoDB running in a container:
 ----
 $ oc new-app -e \
     MONGODB_USER=<username>,MONGODB_PASSWORD=<password>,MONGODB_DATABASE=<database_name>,MONGODB_ADMIN_PASSWORD=<admin_password> \
-    centos/mongodb-26-centos7
+ifdef::openshift-enterprise[]
+    registry.access.redhat.com/rhscl/mongodb-26-rhel7
+endif::[]
+ifdef::openshift-origin[]
+     centos/mongodb-26-centos7
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: Minor-est of nits, but there's an extra space right before the `centos/` here.

**Comment by adellape** [comment]
@ahardin-rh Looks good in a local build.

**Comment by ahardin-rh** [comment]
Extra space removed.

**Comment by ahardin-rh** [comment]
**revhistory note:**

(for Using Images --> DB Images)

    Updated the commands for creating new databases for link:../using_images/db_images/mysql.html[MySQL], link:../using_images/db_images/postgresql.html[PostgresSQL], and link:../using_images/db_images/mongodb.html[MongoDB].

**Pull request merged**